### DISCUSSION TITLE
Question: best practice to make full dialog scrollable.

### DISCUSSION THREAD

**Discussion by latin-1** [ariakit/ariakit#1335] [Questions] [answered]
Bootstrap Model:

https://user-images.githubusercontent.com/44946699/167283876-25ba4792-8664-4562-a323-a368edda5324.mp4

**Comment by diegohaz** [comment, marked as answer]
Make the backdrop element scrollable and remove any `height`/`max-height`/`overflow`/`position: fixed` property from the dialog.

You can pass props such as `className` and `style` to the backdrop element with `<Dialog backdropProps={{ ... }}>` or style the backdrop globally using the `[data-backdrop]` selector.
